server/osx/server_osx.py

```diff
@@ -246,11 +246,13 @@
     'help': 114,
     'home': 115,
     'pgup': 116,  # pageup
+    'pageup': 116,  # pageup
```

Comment:
@calmofthestorm It appears that all of the short hands of the keys must be typed out on this map as well, as in the dragonfly documentation lists various short hands. Https://dragonfly2.readthedocs.io/en/latest/actions.html#key-names

It doesn't seem like a very good solution duplicating all of the short hands between the different servers, what do you think? Should we have some kind of map that all the three servers use that handles the short hands?

Reply:
Yeah I think it would be best to pull this out into a separate file that is shared with all three servers (or possibly have a per-server text/json format?)

That said, if duplicating the code gets the job done most easily I can live with it.

Reply:
By which I mean, I don't think this PR needs to be blocked on that improvement -- we can open an issue instead. Feel free to assign it to me if you don't want to do it -- I'm hoping to take a few days toward the end of the year and look at the backlog of things I said I'd do but haven't had time:-)